src/main/java/com/github/virtualgamers/hrtg/rest/DatabaseLoader.java

```diff
@@ -0,0 +1,24 @@
+package com.github.virtualgamers.hrtg.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import com.github.virtualgamers.hrtg.core.repository.PartyRepository;
+import com.github.virtualgamers.hrtg.core.repository.UserRepository;
+
+/**
+ * root passw0rd
```

Comment:
lol was this on purpose?

Reply:
Must have pasted this here before the Props file was created as a mental note of what I set the password to.